### ISSUE TITLE
neuralop.utils.spectrum_2d() uses deprecated torch API

### ISSUE THREAD

**Issue by profPlum** [neuraloperator/neuraloperator#516]
Hi, your code uses the old torch.fft2 api (using normalize=False) I believe the updated version should look like this:
Notice the if/else normalize statement that is where the change is.

```
# Define the function to compute the spectrum
def spectrum_2d(signal, n_observations, normalize=True):
    """This function computes the spectrum of a 2D signal using the Fast Fourier Transform (FFT).

    Paramaters
    ----------
    signal : a tensor of shape (T * n_observations * n_observations)
        A 2D discretized signal represented as a 1D tensor with shape
        (T * n_observations * n_observations), where T is the number of time
        steps and n_observations is the spatial size of the signal.

        T can be any number of channels that we reshape into and
        n_observations * n_observations is the spatial resolution.
    n_observations: an integer
        Number of discretized points. Basically the resolution of the signal.

    Returns
    --------
    spectrum: a tensor
        A 1D tensor of shape (s,) representing the computed spectrum.
    """
    T = signal.shape[0]
    signal = signal.view(T, n_observations, n_observations)

    if normalize:
        # normalization is no longer applied by default, needs to be specified explicitly
        signal = torch.fft.fft2(signal, norm='ortho')
    else:
        signal = torch.fft.rfft2( # norm='backward' means no normalization
            signal, s=(n_observations, n_observations), norm='backward'
        )

    # 2d wavenumbers following PyTorch fft convention
    k_max = n_observations // 2
    wavenumers = torch.cat(
        (
            torch.arange(start=0, end=k_max, step=1),
            torch.arange(start=-k_max, end=0, step=1),
        ),
        0,
    ).repeat(n_observations, 1)
    k_x = wavenumers.transpose(0, 1)
    k_y = wavenumers

    # Sum wavenumbers
    sum_k = torch.abs(k_x) + torch.abs(k_y)
    sum_k = sum_k

    # Remove symmetric components from wavenumbers
    index = -1.0 * torch.ones((n_observations, n_observations))
    k_max1 = k_max + 1
    index[0:k_max1, 0:k_max1] = sum_k[0:k_max1, 0:k_max1]

    spectrum = torch.zeros((T, n_observations))
    for j in range(1, n_observations + 1):
        ind = torch.where(index == j)
        spectrum[:, j - 1] = (signal[:, ind[0], ind[1]].sum(dim=1)).abs() ** 2

    spectrum = spectrum.mean(dim=0)
    return spectrum
```

**Comment by dhpitt** [comment]
Hi @profPlum , thank you for raising this issue. None of our core models or layers currently use spectrum_2d; it's an old utility for visualization. Thank you for bringing this to my attention nonetheless. I'll make a fix shortly

**Comment by profPlum** [comment]
Ok just double check that norm = “ortho“ is the one that you want when normalize=True ChatGPT told me this setting matched the old behavior but it might be wrong.

**Comment by dhpitt** [comment]
Closed with #519
